tempto-core/src/main/java/io/prestosql/tempto/internal/listeners/TestNameGroupNameMethodSelector.java

```diff
@@ -65,14 +67,21 @@
     public TestNameGroupNameMethodSelector()
     {
         this(getListSystemProperty(TEST_NAMES_TO_RUN_PROPERTY),
+                getListSystemProperty(TEST_NAMES_TO_EXCLUDE_PROPERTY),
                 getListSystemProperty(TEST_GROUPS_TO_RUN_PROPERTY),
                 getListSystemProperty(TEST_GROUPS_TO_EXCLUDE_PROPERTY),
                 new TestMetadataReader());
     }
 
-    public TestNameGroupNameMethodSelector(Optional<Set<String>> testNamesToRun, Optional<Set<String>> testGroupsToRun, Optional<Set<String>> testGroupsToExclude, TestMetadataReader testMetadataReader)
+    public TestNameGroupNameMethodSelector(
+            Optional<Set<String>> testNamesToRun,
+            Optional<Set<String>> testNamesToExclude,
```

Comment:
Is there a semantics difference between `Optional.empty()` and `Optional.of(Set.of())` for `testNamesToExclude`?

Seems none to me.

Same for `testGroupsToExclude`, so you can have a prep commit removing unnecessary Optional usage and then another one adding `testNamesToExclude`